waterlog/src/Components/Usage.js

```diff
@@ -87,16 +87,25 @@ class Usage extends Component {
       monthlyCostForecastError,
       placeholderLoading,
       placeholderError,
+      dailyUsage,
+      dailyCost,
+      monthlyUsage,
+      monthlyCost,
+      seasonUsage,
+      seasonalCost,
+      forecastDaily,
+      forecastMonthly,
       seasonalLoading } = this.props;
 
-    if (dailyError || monthlyError
+    if ((dailyError || monthlyError
```

Comment:
Please make this more readable 
I have dm's you a possible fix

Reply:
Done